File: 1st/Sunhee/JavaScript/문자열 내 p와 y의 개수.js
function solution(s){
    var count_p = 0;
    var count_y = 0;

    for(var i=0; i<s.length; i++){
        if(s[i] === 'p' || s[i] === 'P'){
            count_p++;
        }
        if(s[i] === 'y' || s[i] === 'Y'){
            count_y++;
        }
    }
    return (count_p === count_y) ? true : false;
    
}

File: 4st/Hani/programmer/문자열내 p와 y의 개수.java
class Solution {
    boolean solution(String s) {
        boolean answer = true;
        int countP = 0;
        int countY = 0;
        
        for (int i = 0; i < s.length(); i++) {
            if (Character.toLowerCase(s.charAt(i)) == 'p') {
                countP++;
            }
            else if (Character.toLowerCase(s.charAt(i)) == 'y') {
                countY++;
            }
        }
        
        if (countP != countY) {
            answer = false;
        }

        return answer;
    }
}